conanfile.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Conan recipe package for BehaviorTree.CPP
+"""
+from conans import ConanFile, CMake, tools
+
+
+class BehaviorTreeConan(ConanFile):
+    name = "BehaviorTree.CPP"
+    license = "MIT"
+    url = "https://github.com/BehaviorTree/BehaviorTree.CPP"
+    author = "Davide Faconti <davide.faconti@gmail.com>"
+    topics = ("conan", "behaviortree", "ai", "robotics", "games", "coordination")
+    description = "This C++ library provides a framework to create BehaviorTrees. It was designed to be flexible, easy to use and fast."
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "zmq": [True, False]}
+    default_options = {"shared": False, "zmq": True}
+    generators = "cmake"
+    exports = "LICENSE"
+    exports_sources = ("cmake/*", "include/*", "src/*", "3rdparty/*", "CMakeLists.txt")
+
+    def requirements(self):
+        """Install ZMQ when required
+        """
+        if self.options.zmq:
+            self.requires("cppzmq/4.3.0@bincrafters/stable")
```

Comment:
Shouldn't this have some effect on the build? Like some option USE_ZMQ to CMakeLists?

Reply:
well, find_package for ZMQ is not [mandatory](https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/CMakeLists.txt#L28) so we could build BehaviorTree without cppzmq. I think it could be an option for tiny version of package.